### ISSUE TITLE
Compilation failure from nix 0.7, required by mortal 0.1.5, by linefeed 0.5.4

### ISSUE THREAD

**Issue by AlbertoGP** [murarth/ketos#65]
Hi, trying to compile the git repo with **rust stable 1.39**.
```
error[E0432]: unresolved import `self::consts`
  --> ~/.cargo/registry/src/github.com-1ecc6299db9ec823/nix-0.7.0/src/sys/ioctl/platform/linux.rs:34:15
   |
34 | pub use self::consts::*;
   |               ^^^^^^ could not find `consts` in `self`
...
```

`nix 0.7` is required by `mortal 0.1.5`, which is in turn required by `linefeed 0.5.4`.

Switching to `linefeed 0.6` in `Cargo.toml` solves the issue.

**Comment by murarth** [comment]
I've just updated all dependencies and published version 0.12.0. Does this resolve your issue?

**Comment by AlbertoGP** [comment]
Sure, it now compiles cleanly from a fresh clone. Thanks!
